### ISSUE TITLE
Disable CallKit on iOS

### ISSUE THREAD

**Issue by focalsolution** [skrafft/react-native-jitsi-meet#280]
**Description**
Recently, I got a reject from Apple state that my application is using **Callkit** which is not allow in **China**.
Is there any option to disable this library from react-native-jitsi-meet.

**Environment**
iOS: 14.3
react-native: 0.62.2
react-native-jitsi-meet: 2.1.1

**Comment by G10dRafaFan** [comment]
Hi @focalsolution,
Facing same issue are you able to resolved the issue?

**Comment by rizbud** [comment]
Facing same issue. Are you guys able to solve this issue?

**Comment by focalsolution** [comment]
No, I cannot find a way to disable the Callkit. Therefore, I simply end up not distribute my app in Chaina. ;)

**Comment by rizbud** [comment]
@skrafft can you help to fix this issue?

**Comment by G10dRafaFan** [comment]
@rizbud , didn't find the solution to disable the callkit.

**Comment by rizbud** [comment]
> @rizbud , didn't find the solution to disable the callkit.

Then do you have any solution to publish the app to the Apple App Store?

**Comment by G10dRafaFan** [comment]
@rizbud we have not published on App Store for china
